api/catalog/api/controllers/elasticsearch/utils.py

```diff
@@ -129,10 +198,18 @@ def post_process_results(
                 return results
 
             s = s[start:end]
+            logger.info(f"executing additional filler query depth={depth}")
```

Comment:
I initially thought this was a typo and `filler` was meant to be `filter`, but I realize now this is probably using "fill" to refer to the process of `"continue accruing additional results until the requested page_size of validated results is reached"` described above. Is that correct?

Reply:
Yes! I was struggling to find a word to describe the process. "backfill" didn't seem right to me, but maybe it technically is what we're doing: https://www.thefreedictionary.com/backfill

> to refill an excavated trench, esp (in archaeology) at the end of an investigation

More or less what we're doing here lol!